build.go

```diff
@@ -46,26 +46,43 @@ func Build(procs ProcMgr, logger scribe.Emitter) packit.BuildFunc {
 		}
 		layer.Launch = true
 
+		httpdConfPath, httpdOk := os.LookupEnv("PHP_HTTPD_PATH")
```

Comment:
@sophiewigmore I think we can simplify this by changing `os.LookupEnv` to `os.Getenv`, as we treat the empty and unset conditions equally. We can keep the existing tests (which test both empty and unset) but simplify the code.